test/tests/userobjects/openmc_tally_editor/tests

```diff
@@ -0,0 +1,132 @@
+    expect_err = "In attempting to set tally nuclides in the 'tally_editor_uo' UserObject, OpenMC reported:\n\n"
+                 "Nuclide 'fake' is not present in library."
+    requirement = "The system shall error if trying to add a nuclide not accessible in the cross section library."
+    required_objects = 'OpenMCCellAverageProblem'
+  []
+  [nuclides]
+    type = CheckFiles
+    input = tally_param_checks.i
+    check_files = tallies.out
+    file_expect_out ='U238'
+    cli_args = 'UserObjects/tally_editor_uo/nuclides="U238"'
+    requirement = "The system shall confine the tally to a particular nuclide. This is compared by selecting just one of the existing nuclides, and comparing to the sum of the total nuclides."
+    required_objects = 'OpenMCCellAverageProblem'
+  []
+  [scatter_tally]
+    type = CheckFiles
+    input = scatter.i
+    check_files = tallies.out
+    requirement = 'Ensure that scores specified by a tally editor UO are present in the tally output'
+    file_expect_out = 'Scattering Rate'
+    required_objects = 'OpenMCCellAverageProblem'
+  []
+  [absorption_u238]
+    type = CheckFiles
+    input = nuclide_absorption.i
+    check_files = tallies.out
+    requirement = 'Ensure that scores specified by a tally editor UO are present in the tally output'
+    file_expect_out = 'U238\n   Absorption Rate'
+    required_objects = 'OpenMCCellAverageProblem'
+  []
+  [invalid_score]
+    type = RunException
+    input = tally_param_checks.i
+    cli_args = 'UserObjects/tally_editor_uo/scores="invalid"'
+    expect_err = 'Invalid tally score "invalid". See the docs for details:'
+    requirement = 'The system shall error if trying to add a score with an invalid type.'
+    required_objects = 'OpenMCCellAverageProblem'
+  []
+  [add_cell_filter]
+    type = CheckFiles
+    input = add_filter.i
+    check_files = tallies.out
+    requirement = 'Ensure that filters specified by a tally editor UO are present in the tally output'
+    file_expect_out = 'Cell'
+    required_objects = 'OpenMCCellAverageProblem'
+  []
+  [add_material_filter]
+    type = CheckFiles
+    input = add_filter.i
+    check_files = tallies.out
+    cli_args = 'UserObjects/new_filter/filter_type="material"'
+    requirement = 'Ensure that filters specified by a tally editor UO are present in the tally output'
+    file_expect_out = 'Material'
+    required_objects = 'OpenMCCellAverageProblem'
+  []
+  [add_universe_filter]
+    type = CheckFiles
+    input = add_filter.i
+    check_files = tallies.out
+    cli_args = 'UserObjects/new_filter/filter_type="universe"'
+    requirement = 'Ensure that filters specified by a tally editor UO are present in the tally output'
+    file_expect_out = 'Material'
+    required_objects = 'OpenMCCellAverageProblem'
+  []
+  [invalid_filter_type]
+    type = RunException
+    input = add_filter.i
+    cli_args = 'UserObjects/new_filter/filter_type="invalid"'
+    expect_err = 'Invalid option "invalid" in MooseEnum.'
```

Comment:
We don't need to have this test; MOOSE handles error checking for invalid enums, now that we use the MooseEnum type we get this for free. I'll remove it in the commit I push.

Reply:
Nice! Removing...